python/cugraph/structure/graph_primtypes_wrapper.pyx

```diff
@@ -220,11 +220,27 @@ def _degree_csr(offsets, indices, x=0):
     return vertex_col, degree_col
 
 
-def _degree(input_graph, x=0):
+def _degree(input_graph, x=0, mnmg=False):
     transpose_x = { 0: 0,
                     2: 1,
                     1: 2 }
 
+    if mnmg:
```

Comment:
This might be a little cleaner (and make the calling code more self-documenting) as just a separate method: `def _mg_degree(input_graph, x=0)`

Reply:
updated